extensions/cocostudio/src/cocostudio/SGUIReader.cpp

```diff
@@ -1470,13 +1470,13 @@ Widget* WidgetPropertiesReader0300::widgetFromBinary(CocoLoader* cocoLoader, stE
             customJsonDict.Parse<0>(customProperty);
             if (customJsonDict.HasParseError())
             {
-                AXLOG("GetParseError %d\n", customJsonDict.GetParseError());
+                AXLOGD("GetParseError {}\n", customJsonDict.GetParseError());
```

Comment:
Change `AXLOGD("GetParseError {}\n", customJsonDict.GetParseError());` to `AXLOGD("GetParseError {}\n", static_cast<int>(customJsonDict.GetParseError()));` will fix winuwp compile errors, and remember change all occurrences.

Reply:
@halx99 
Once again: Any hints?

`build / winuwp (pull_request) Failing after 30m`